access-control-app/src/cmr/access_control/api/routes.clj

```diff
@@ -368,7 +370,11 @@
 
         (GET "/"
              {ctx :request-context params :params}
-             (get-permissions ctx params)))
+             (get-permissions ctx params))
+
+        (POST "/"
+              {ctx :request-context headers :headers body :body}
+              (get-permissions ctx (walk/keywordize-keys (codec/form-decode (slurp body))))))
```

Comment:
No need to parse the query string yourself. Ring middleware has already put the query parameters in the `:params`.